backend/geonature/app.py

```diff
@@ -171,7 +171,7 @@ def load_current_user():
         def set_sentry_context():
             if "FLASK_REQUEST_ID" in request.environ:
                 set_tag("request.id", request.environ["FLASK_REQUEST_ID"])
-            if g.current_user:
+            if hasattr(g, "current_user") and g.current_user:
```

Comment:
`if hasattr(g, "current_user") and g.current_user.is_authenticated`
-> il peut y avoir un utilisateur non connecté (https://flask-login.readthedocs.io/en/latest/#anonymous-users), du coup le g.current_user.id_role peut faire bugger juste en dessous